variables.tf

```diff
@@ -50,15 +50,24 @@ variable "alerting_email" {
 }
 
 variable "memory" {
-  description = "The amount of memory (in MiB) used by Waggle Dance task."
-  type        = "string"
-  default     = "4096"
+  description = <<EOF
+The amount of memory (in MiB) used by Waggle Dance task.
```

Comment:
Isn't this more accurate:

The amount of memory (in MiB) that will be made available for use by the Waggle Dance task.

Reply:
And below you say "container" so should the above be "container" instead of "task"? Go for whichever term is more accurate and then make sure the documentation is consistent.

Reply:
updated in other PR